### PR TITLE
Open the picker on click if it wasn't already open

### LINKED ISSUE #717: dropdown menu(datepicker) will not show up again once click the parent container scrollbar
There was a ticket created for this issue: https://github.com/YouCanBookMe/react-datetime/issues/283

The steps to reproduce:
(1) click on the datetime field, the datepicker show up
(2) before click on any date on datepicker,   click on the scrollbar.  the datepick will be closed automatically.
(3) now click on the datetime field, the datepicker will not show up 

I noticed that when click on the scrollbar in step (2), the datetime field still on focus.

### PR DESCRIPTION
**Pull request by arqex**
This ticket fixes #717 

```
[ ] I have not included any built dist files (us maintainers do that prior to a new release)
[x ] I have added tests covering my changes
[x ] All new and existing tests pass
[ ] My changes required the documentation to be updated
  [ ] I have updated the documentation accordingly
  [ ] I have updated the TypeScript 1.8 type definitions accordingly
  [ ] I have updated the TypeScript 2.0+ type definitions accordingly
```

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ class App extends React.Component {
 
 	render() {
 		return (
-			<div>
+			<div style={{height: 2000}}>
```
Comment (onlined) [inline src/App.js:17]: Why is this needed?

Comment (arqex) [reply src/App.js:17]: I needed the playground to scroll to reproduce the issue, and then I've just forgotten about remove it. This happens to me all the time with the playground :D

Comment (arqex) [reply src/App.js:17]: Hey @onlined, do you want to be member of the project? After seeing your PRs and the detailed reviews you made I'd be glad to have you aboard. (I could have reviews on my PRs too!)

Comment (onlined) [reply src/App.js:17]: It would be great, thank you very much!

Comment (arqex) [reply src/App.js]: Nice1 I've sent an invite for you :)

**Comment by LeoFengNY** [comment]
It is still not working after updating from the version "2.16.3" to "^3.0.4". I tested in all the major browsers including Chrome, IE and firefox.(the ticket #717 )

**Pull request merged**